### DISCUSSION TITLE
Marketplace for the Plugin System

### DISCUSSION THREAD

**Discussion by MnSx3** [helix-editor/helix#5557] [Ideas]
I believe it would be beneficial to consider a marketplace for the plugin system. 
One major drawback of Neovim is the lack of organization and inefficiency due to the absence of a marketplace for plugins. 

How do you think we should go about implementing this?

**Comment by goyalyashpal** [comment]
One detail, whenever it comes to fruition, "central repository" or smth like that will be a more apt name.

**Reply by nik-rev** [reply]
Unless we also decide to add Helixcoins to purchase Helix addons. Then we can truly call it the marketplace :)
